.github/workflows/continuous-integration.yaml

```
@@ -9,6 +9,33 @@ on:
       - 'alpha-release/**'
 
 jobs:
+  autopep8:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: autopep8
+        id: autopep8
+        uses: peter-evans/autopep8@v1
+        with:
+          # ignore E402 rule to avoid changes that cause circular imports
+          args: --exit-code --recursive --in-place --aggressive --aggressive --ignore E402 .
```

To use line lengths of 100, as found in the Google Java code style guide which we use for our Java code. This removes the needs for many changes to the existing codebase.
```suggestion
          args: --exit-code -r --in-place -a -a --ignore E402 --max-line-length 100 .
```

---

I'll make that change once the rules have been confirmed- this was mainly like this to try to help with readability/understanding during the initial PR. If you want it sooner than at the very end, i can make the changes.